contrib/cirrus/integration_test.sh

```diff
@@ -37,11 +37,16 @@ case "$SPECIALMODE" in
                 -o CheckHostIP=no $GOSRC/$SCRIPT_BASE/rootless_test.sh ${TESTSUITE}
         ;;
     cgroupv2)
+        # FIXME: use the package once all the fixes are in a release
+        # yum install -y crun
+        setenforce 0
+        yum builddep -y crun
+        (git clone --depth=1 https://github.com/containers/crun && cd crun && ./autogen.sh && ./configure --prefix=/usr && make -j4 && make install)
+        export OCI_RUNTIME=/usr/bin/crun
         make
         make install PREFIX=/usr ETCDIR=/etc
         make test-binaries
-        echo "WARNING: Integration tests not yet ready for cgroups V2"
-        #TODO: make local${TESTSUITE}
+        make local${TESTSUITE}
```

Comment:
Are you intending to support running of the system-tests?  If so, please add to `.cirrus.yml`:

```yaml
special_testing_cgroupv2_task
    ...
    system_test_script: '$SCRIPT_BASE/system_test.sh |& ${TIMESTAMP}'
```

Reply:
I've not yet looked into them, I'd expect them to fail at this point.  We can add them later, once we are sure they work with a different OCI runtime